test/Microsoft.AspNetCore.Mvc.Core.Test/Builder/MvcAreaRouteBuilderExtensionsTest.cs

```diff
@@ -227,6 +228,48 @@ public void MapAreaRoute_DoesNotReplaceValuesForAreaIfAlreadyPresentInConstraint
                 });
         }
 
+        [Fact]
+        public void MapAreaRoute_DotCheck()
```

Comment:
`MapAreaRoute_UsesPassedInAreaNameAsIs` (or something like that. (DotCheck doesn't necessarily say what this means).